templates/cluster/gcp-standalone-cp/values.yaml

```diff
@@ -0,0 +1,74 @@
+  subnet: "" # @schema description: A reference to the subnetwork to use for this instance. If not specified, the first subnetwork retrieved from the Cluster Region and Network is picked; type: string; patternProperties: {"^[a-z]$": {"type": "string"}}
+  providerID: "" # @schema description: The unique identifier as specified by the cloud provider; type: string
+  imageFamily: "" # @schema description: The full reference to a valid image family to be used for this machine; type: string
+  image: "" # @schema description: The full reference to a valid image to be used for this machine. Takes precedence over imageFamily; type: string
+  additionalLabels: {} # @schema description: Additional set of labels to add to an instance, in addition to the ones added by default by the GCP provider. If both the GCPCluster and the GCPMachine specify the same tag name with different values, the GCPMachine's value takes precedence; type: object; additionalProperties: true; patternProperties: {"^[a-z]$": {"type": "string"}}
+  publicIP: true # @schema description: PublicIP specifies whether the instance should get a public IP. Set this to true if you don't have a NAT instances or Cloud Nat setup; type: boolean
+  additionalNetworkTags: [] # @schema description: A list of network tags that should be applied to the instance. These tags are set in addition to any network tags defined at the cluster level or in the actuator; type: array; item: string
+  rootDeviceSize: 30 # @schema description: The size of the root volume in GB; type: number; minimum: 1
+  rootDeviceType: pd-standard # @schema description: The type of the root volume; type: string; enum: pd-standard,pd-ssd,pd-balanced,hyperdisk-balanced;
+  rootDiskEncryptionKey:
+  serviceAccount: # @schema description: The service account email and which scopes to assign to the machine; type: object
+    email: "default" # @schema description: Email address of the service account; type: string
+    scopes: # @schema description: The list of scopes to be made available for this service account; type: array; item: string
+      - "compute.CloudPlatformScope"
+  ipForwarding: Enabled # @schema description: IPForwarding allows this instance to send and receive packets with non-matching destination or source IPs. This is required if you plan to use this instance to forward routes. Defaults to Enabled; type: string; enum: Enabled,Disabled
+
+# K0s parameters
+k0s: # @schema description: K0s parameters; type: object
+  version: v1.31.5+k0s.0 # @schema description: K0s version; type: string
+  api: # @schema description: Kubernetes API server parameters; type: object; additionalProperties: object
+    extraArgs: {} # @schema description: Map of key-values (strings) for any extra arguments to pass down to Kubernetes api-server process; type: object; additionalProperties: true; patternProperties: {"^[a-z]$": {"type": "string"}}
```

Comment:
I am not aware, can't the args have capital letters?

Reply:
Actually, it can. I copy-pasted this rule from another template, but it should be fixed everywhere. Thanks for noticing.